app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<section class="flex-center column public-recipes recipes">
+  <div class="max-width flex-center">
+    <div class="row food-heading">
+     
+      <%= button_to "Add Recipe", new_recipe_path, method: :get, class:"btn-bg-red-big" %>
+    </div>
+  </div>
+
+  <p class="notice"><%= notice %></p>
+  <p class="btn-text txt-list-recipes">List Recipes</p>
+  <ul class="max-width flex-center column">  
+    <% @recipes.each do |recipe| %>
+      <li class="card row-wrap public-recipe">  
+        <div class="head">
+          <a href=<%= "recipes/#{recipe.id}" %> class="no-styling">
+            <h3><%= recipe.name %></h3>
+          </a>
+           <a href=<%= "recipes/#{recipe.id}" %> class="no-styling text-description">
+          <p class="text-description"><%= recipe.description %></p>
+        </a>
+        </div>
```

Comment:
- [ ] Please make sure that your recipes list page layout design matches the provided wireframe below :point_down: 
  
  ### Your design
  ![image](https://github.com/TaiwoEnoch/FoodyOnRails/assets/92303891/5bf297f5-d4c3-4cd4-b841-1c36c9ff6478)
  
  ### Provide wireframe
  ![image](https://github.com/TaiwoEnoch/FoodyOnRails/assets/92303891/2b465f41-0429-4ce9-8774-18b707e87ef0)